supporting_content/dimensions/chargefrequency.md

```diff
@@ -0,0 +1,39 @@
+# Column: ChargeFrequency
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                                                                                                                                            |
+| --------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AWS       | CUR                      | Bill_BillType                                                                                                                                     |
+| GCP       | Big Query Billing Export | NA (No column name has been currently identified that performs partially or fully the function that the proposed dimension ChargeFrequency would) |
+| Microsoft | Cost details             | Frequency                                                                                                                                         |
+                                             
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Data set                 | Example Value                |
+| --------- | ------------------------ | ---------------------------- |
+| AWS       | CUR                      | Anniversary,Purchase,Refund  |
+| GCP       | Big Query Billing Export | NA                           |
+| Microsoft | Cost details             | Recurring,Monthly,UsageBased |
```

Comment:
I didn't think we had "Monthly", but I do know that was the original intent, so I wouldn't be surprised if someone snuck it in.
```suggestion
| Microsoft | Cost details             | OneTime, Recurring, Monthly, UsageBased |
```